gitlab/tests/compose/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '2'
 services:
   gitlab:
-    image: gitlab/gitlab-ce:${GITLAB_VERSION}
+    image: gitlab/gitlab-ce:latest
```

Comment:
pinning to latest is a bad idea cause it causes master to break due to external circumstances. You can pin to 13 if you want